lib/core/Axios.js

```diff
@@ -157,8 +157,7 @@ utils.forEach(['post', 'put', 'patch'], function forEachMethodWithData(method) {
     };
   }
 
-  Axios.prototype[method] = generateHTTPMethod();
-
+  Axios.prototype[method] = generateHTTPMethod(false);
```

Comment:
The omission of the default value is done to save a few bytes for the client's build. That just should be described in the JSDoc comment if this needs clarification.

Reply:
I suspected that but to try to keep the code standardized, I thought of putting false. But in this case, to save bytes, I think it's valid